### PR TITLE
Develop1

### PR DESCRIPTION
**Pull request by yuliiayedynak**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,43 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
```
Comment (viktoria-rybenchuk) [inline app/main.py]: Place each argument on a new line including self for better readability

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,43 @@
...
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
```
Comment (viktoria-rybenchuk) [inline app/main.py]: the same here

**Review by viktoria-rybenchuk — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 2 above)
Comment (yuliiayedynak) [reply app/main.py]: дякую! нібито виправила

**Review by viktoria-rybenchuk — APPROVED** [review]
GJ!

**Pull request closed without merging**